sky/task.py

```diff
@@ -550,12 +557,6 @@ def add_storage_mounts(self) -> None:
                         mnt_path: blob_path,
                     })
                 elif store_type is storage_lib.StoreType.GCS:
-                    # Remember to run `gcloud auth application-default login`
```

Comment:
Any change made this no longer needed?

Reply:
These lines are not effective as the file_mounts will be executed before the setup commands are executed. Instead, I changed the `_GSUTIL` in `cloud_store.py` to `_GSUTIL = (f'GOOGLE_APPLICATION_CREDENTIALS='{gcp.DEFAULT_GCP_APPLICATION_CREDENTIAL_PATH} gsutil')`, so that the credential path is added to the environment variable.